src/Compilers/VisualBasic/Portable/Utilities/FeatureUtils.vb

```diff
@@ -0,0 +1,83 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+Imports System.Runtime.CompilerServices
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
```

Comment:
So [these](https://github.com/dotnet/roslyn/blob/master/src/Compilers/VisualBasic/Portable/Parser/ParserFeature.vb) are also in the wrong location.